spec/request/api_open_library_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'json'
+
+RSpec.describe 'Request to /api/open_library/:isbn', type: :request do
+  describe 'when using ISBN:2' do
```

Comment:
the `action`  statement after `describe` define and method that you use in the test

look this 
```ruby
describe PostsController do
  describe action 'GET #index' do # `action` after describe define the method that you make testing
    describe example 'without errors' do # `example` after describe define that is an example of the action
        default_example # You can select an example to be marked as default in your action

        before do
            get :index
        end

        # ... your tests
    end
  end
end

```
```suggestion
  describe action 'Get book info' do
```